src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const populations = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+const populationList = document.querySelector('.list');
+
+const sum = Array.from(populations).reduce(
+  (acc, cur) => acc + Number(cur.innerText.replaceAll(',', '')),
+  0,
+);
+
+totalPopulation.innerText = sum.toLocaleString('en-US');
+
+averagePopulation.innerText = Math.round(
+  sum / populationList.children.length,
```

Comment:
Ensure that `populationList.children.length` accurately reflects the number of population elements. If `populationList` contains elements other than those with the class 'population', this could lead to an incorrect average calculation. Consider using `populations.length` instead if it better represents the number of population entries.